### PR TITLE
Dashboard

### PR DESCRIPTION
**Pull request by wellitonSouza**

### CODE REVIEW

**Review by thiagodpf — APPROVED** [review]

File: source/locale/pt_BR/LC_MESSAGES/components-and-apis.po [hunk 1]
```diff
@@ -400,138 +408,149 @@ msgstr ""
 "expostos** e **endpoints de componentes**. Veja mais sobre como usar as "
 "APIs em :doc:`./using-api-interface`."
 
-#: ../../source/components-and-apis.rst:129
+#: ../../source/components-and-apis.rst:133
 msgid "Exposed endpoints"
-msgstr "*Endpoint* exposto"
+msgstr "Endpoints expostos"
```
Comment (mprevide) [inline source/locale/pt_BR/LC_MESSAGES/components-and-apis.po:413]: Put between `*` when is using a  english word in a pt traslantion.

File: source/locale/pt_BR/LC_MESSAGES/components-and-apis.po [hunk 2]
```diff
@@ -400,138 +408,149 @@ msgstr ""
...
 
 #: ../../source/components-and-apis.rst
 msgid "Exposed endpoint"
-msgstr "*Endpoint* exposto"
+msgstr ""
```
Comment (mprevide) [inline source/locale/pt_BR/LC_MESSAGES/components-and-apis.po:417]: Add translation

File: source/locale/pt_BR/LC_MESSAGES/components-and-apis.po [hunk 3]
```diff
@@ -400,138 +408,149 @@ msgstr ""
...
 
 #: ../../source/components-and-apis.rst
 msgid "by the API gateway"
-msgstr "pelo API *gateway*"
+msgstr ""
```
Comment (mprevide) [inline source/locale/pt_BR/LC_MESSAGES/components-and-apis.po:421]: Add translation

File: source/locale/pt_BR/LC_MESSAGES/components-and-apis.po [hunk 4]
```diff
@@ -400,138 +408,149 @@ msgstr ""
...
 
 #: ../../source/components-and-apis.rst
 msgid "Endpoint"
 msgstr "*Endpoint*"
 
 #: ../../source/components-and-apis.rst
 msgid "Needs"
-msgstr "Precisa de"
+msgstr ""
```
Comment (mprevide) [inline source/locale/pt_BR/LC_MESSAGES/components-and-apis.po:429]: Add translation

File: source/locale/pt_BR/LC_MESSAGES/using-web-interface.po [hunk 5]
```diff
@@ -337,7 +337,7 @@ msgstr "Atualização de Firmware"
 msgid ""
 "During the lifetime of a device, you may need to update control software "
 "(firmware) to correct some issues you encounter while using it, or even "
-"add new features. Dojot currently supports the firmware upgrade procedure"
+"add new features. dojot currently supports the firmware upgrade procedure"
```
Comment (mprevide) [inline source/locale/pt_BR/LC_MESSAGES/using-web-interface.po:340]: `Dojot` instead of `dojot`

File: source/using-web-interface.rst [hunk 6]
```diff
@@ -189,7 +189,7 @@ Firmware update
 ---------------
 
 During the lifetime of a device, you may need to update control software (firmware) to correct some issues you encounter while using it, or even add new features.
-Dojot currently supports the firmware upgrade procedure via the LwM2M communication protocol. For details regarding the procedure for integrating with your device please check the LwM2M protocol specification. If your device communicates via this protocol and has the firmware update procedure in place, you can follow the steps below to update your device version.
+dojot currently supports the firmware upgrade procedure via the LwM2M communication protocol. For details regarding the procedure for integrating with your device please check the LwM2M protocol specification. If your device communicates via this protocol and has the firmware update procedure in place, you can follow the steps below to update your device version.
```
Comment (mprevide) [inline source/using-web-interface.rst:192]: Dojot

**Review by mprevide — CHANGES_REQUESTED** [review]
I left a few comments

File: source/using-web-interface.rst (hunk 6 above)
Comment (wellitonSouza) [reply source/using-web-interface.rst]: I did as requested changes

**Review by mprevide — APPROVED** [review]

**Review by thiagodpf — APPROVED** [review]

**Pull request merged**